### PR TITLE
[OKU-26] FEAT:Ability to reject a review

### PR DESCRIPTION
**Pull request by twentymurial33**
Summary of changes
Change Type
*Put an x in all the boxes that apply

[] Bugfix (non-breaking change which fixes an issue)
[X]Feature (non-breaking change which adds functionality)
[] Breaking change (fix or feature that would cause existing functionality to change)
Linear issue link
https://linear.app/oku-no-kumo/issue/OKU-26/be-ability-to-reject-a-review

Wiki link related to this ticket

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -4,7 +4,7 @@ generator client {
 }
 
 datasource db {
-  provider = "mysql"
+  provider = "postgresql"
```
Comment (enricodelarosa) [inline prisma/schema.prisma]: Maybe we can make this an env variable? So local dev env can choose their own provider on local? What do you think @idiglove

Comment (enricodelarosa) [reply prisma/schema.prisma]: Will adjust to postgresql on my side

**Review by enricodelarosa — APPROVED** [review]
lgtm!

File: controllers/reviewsController.ts [hunk 2]
```diff
@@ -50,3 +51,32 @@ export const approveReview: RequestHandler = async (
...
+    req: Request,
+    res: Response
+) => {
+    const reviewId: number = Number(req.params.reviewId);
+    const rejectedReview: review = await prisma.review.update({
+        where: { id: reviewId },
+        data: {
+          
+        },
+    });
```
Comment (idiglove) [inline controllers/reviewsController.ts]: shouldnt it have {isApproved: false}?

**Review by idiglove — APPROVED** [review]

File: controllers/reviewsController.ts (hunk 2 above)
Comment (twentymurial33) [reply controllers/reviewsController.ts]: Yes - added the Boolean

File: prisma/migrations/20230426123806_delete_review/migration.sql [hunk 3]
```diff
@@ -0,0 +1,58 @@
+-- CreateTable
+CREATE TABLE "user" (
+    "email" VARCHAR(50) NOT NULL,
```
Comment (idiglove) [inline prisma/migrations/20230426123806_delete_review/migration.sql]: what is this migration for?

Comment (twentymurial33) [reply prisma/migrations/20230426123806_delete_review/migration.sql]: I hadn't run `npx prisma migrate dev`

Comment (enricodelarosa) [reply prisma/migrations/20230426123806_delete_review/migration.sql]: I use npx prisma db push. This way we don't have to deal with migrations. If i remember correctly migrations are used when there's already data in prod and prod is already being used, right @idiglove?

**Pull request merged**